SG14/inplace_function.h

```diff
@@ -224,10 +224,11 @@ class alignas(AlignmentT) inplace_function<RetT(ArgsT...), CapacityT, AlignmentT
 		m_ManagerFctPtr = nullptr;
 	}
 
-	template<size_t OtherCapacityT>
-	void copy(const inplace_function<RetT(ArgsT...), OtherCapacityT>& other)
+	template<size_t OtherCapacityT, size_t OtherAlignment>
```

Comment:
I agree, I just didn't want to batch that change with this one and wanted to keep it consistent. Maybe in a follow up PR.